src/index.js

```diff
@@ -33,6 +33,18 @@ const editorSetup = () => {
 	const editPost = require( '@wordpress/edit-post' );
 
 	editPost.initializeEditor();
+	exposeGlobals();
```

Comment:
In my proofs of concept, I didn't need these globals, but that's because I was not using `wp-scripts` to build the plugin bundle, and instead used [these externals](https://github.com/mkevins/gutenberg-plugin-hello-world/commit/e26b9dccc82e2fd999d710408758f7edd2e1bdcf#diff-11e9f7f953edc64ba14b0cc350ae7b9d) in a new `webpack.config`. I wonder if we can customize the `wp-scripts` for the mobile development environment to do something similar. I _think_ that would work similarly in metro's bundling, but I haven't tried it.